### ISSUE TITLE
Make product design decisions for MDF and Foundry

### ISSUE THREAD

**Issue by ascourtas** [MLMI2-CSSI/foundry#383]
_As a Product Owner, we want to make sure our products are on the right path!_

Establish a timeline and make decisions about design for the following:
- MDF SDK decisions
- MDF -> Foundry rebrand
- Garden metadata and database needs 

Can include design documentation (or maybe that's another ticket) 

```[tasklist]
### Tasks
- [x] Discuss in KJ <> Ari <> Ben meeting
- [ ] Make a visual timeline for us to approve or deny
- [ ] Bring that timeline back to KJ and Ben
- [ ] Meet all 3 of us to finalize Foundry/MDF branding
```

# Assumptions:
1. 
2. 

# Acceptance Criteria
Given..., when..., then...

**Comment by kjschmidt913** [comment]
Does this include Foundry model decisions? Or is this just about the relationship between MDF and foundry design decisions?

**Comment by ascourtas** [comment]
I think this would include the model decisions, because otherwise it's hard to settle on branding for Foundry and MDF? Because it wouldn't make sense to do a big brand shift and then change models afterwards. I'm going to mark this as Not Ready until we finalize the models

**Comment by kjschmidt913** [comment]
Okay. It doesn't mention models in the description of the issue, so I wondered if this was more MDF focused for MDF branding. Got it. Yeah, we have to wait on that

**Comment by ascourtas** [comment]
It is focused on the MDF branding, but I think it would be hard to take action on the branding changes until we decided what was included in the Foundry brand?

If you disagree lmk!

For now, I'll say blocked by https://github.com/MLMI2-CSSI/foundry/issues/397
